models/members.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"context"
+	"time"
+)
+
+type Member struct {
+	ID        string    `json:"_id" bson:"_id,omitempty"`
+	Account   *string   `json:"account_id" bson:"account_id,omitempty"`
+	Group     *string   `json:"group_id" bson:"group_id,omitempty"`
+	Role      string    `json:"role" bson:"role,omitempty"`
+	CreatedAt time.Time `json:"created_at" bson:"created_at,omitempty"`
+}
+
+type MemberPayload struct {
+	Account   *string   `json:"account_id" bson:"account_id,omitempty"`
+	Group     *string   `json:"group_id" bson:"group_id,omitempty"`
+	Role      string    `json:"role" bson:"role,omitempty"`
+	CreatedAt time.Time `json:"created_at" bson:"created_at,omitempty"`
```

Comment:
This should be generated automatically and not a parameter to the `Create` method.

Reply:
Fix, it's generate by DB now